.test-infra/jenkins/job_PostCommit_Python_ValidatesContainer_Dataflow.groovy

```diff
@@ -35,11 +35,13 @@ PostcommitJobBuilder.postCommitJob('beam_PostCommit_Py_ValCont',
       }
 
       // Execute shell command to test Python SDK.
-      // TODO: Parallel the script run with Jenkins DSL or Gradle.
       steps {
-        VALIDATES_CONTAINER_DATAFLOW_PYTHON_VERSIONS.each {
-          pythonVersion = it.replace('.', '')
-          shell('cd ' + commonJobProperties.checkoutDir + " && bash sdks/python/container/run_validatescontainer.sh python${pythonVersion}")
+        gradle {
+          rootBuildScriptDir(commonJobProperties.checkoutDir)
+          tasks(':sdks:python:test-suites:portable:py36:validatesContainer')
```

Comment:
This list will need to be updated with every new Python version.
Shall we update the relevant doc for adding and removing python versions.

Reply:
I suggest we add a constant along these lines
https://github.com/apache/beam/blob/2b8938eced3b5776441246f521a38af5269f916a/sdks/python/test-suites/gradle.properties#L28

and a top-level task like so: 
https://github.com/apache/beam/blob/9f9dcd6e97b29c020f555bcaa992af56108aaa0d/sdks/python/test-suites/dataflow/build.gradle#L58

Reply:
done.